autosklearn/pipeline/components/data_preprocessing/feature_type.py

```diff
@@ -29,22 +31,22 @@
 
 
 class FeatTypeSplit(AutoSklearnPreprocessingAlgorithm):
-    """ This component is used to apply distinct transformations to categorical and
-    numerical features of a dataset. It is built on top of sklearn's ColumnTransformer.
+    """ This component is used to apply distinct transformations to categorical,
+    numerical and text features of a dataset. It is built on top of sklearn's ColumnTransformer.
     """
 
     def __init__(
-        self,
-        config: Optional[Configuration] = None,
-        pipeline: Optional[BasePipeline] = None,
-        dataset_properties: Optional[DATASET_PROPERTIES_TYPE] = None,
-        include: Optional[Dict[str, str]] = None,
-        exclude: Optional[Dict[str, str]] = None,
-        random_state: Optional[Union[int, np.random.RandomState]] = None,
-        init_params: Optional[Dict[str, Any]] = None,
-        feat_type: Optional[Dict[Union[str, int], str]] = None,
-        force_sparse_output: bool = False,
-        column_transformer: Optional[sklearn.compose.ColumnTransformer] = None,
+            self,
```

Comment:
Please do not make such formatting changes. These make it hard for me to review your PR.